website/templates/project/project.mako

```diff
@@ -447,29 +447,80 @@
                 </div>
-                <pre id="citationText" class="formatted-citation"></pre>
-            </div>
+             </div>
+             <div id="citationList">
+                 <div class="panel-body" style="display: none;">
+                     <div data-bind="visible: page() == 'loading'">
+                        <div class="spinner-loading-wrapper">
+                            <div class="ball-scale ball-scale-blue">
+                                <div></div>
+                            </div>
+                            <p class="m-t-sm fg-load-message"> Loading citations...  </p>
+                        </div>
+                     </div>
+                     <div data-bind="visible: page() == 'standard'" style="display: none;">
+                         % if not node['anonymous'] and 'admin' in user['permissions']:
+                             <a data-bind="click: showEditBox" class="pull-right"><i class="glyphicon glyphicon-pencil"></i> Customize</a>
+                         % endif
+                         <div class="m-b-md">
+                             <div class="citation-list">
+                                 <div class="f-w-xl">APA</div>
+                                 <span data-bind="text: apa"></span>
+                                 <div class="f-w-xl m-t-md">MLA</div>
+                                 <span data-bind="text: mla"></span>
+                                 <div class="f-w-xl m-t-md">Chicago</div>
+                                 <span data-bind="text: chicago"></span>
+                             </div>
+                         </div>
+                         <p><strong>Get more citations</strong></p>
+                         <div id="citationStylePanel" class="citation-picker">
+                             <input id="citationStyleInput" type="hidden" />
+                         </div>
+                         <pre id="citationText" class="formatted-citation"></pre>
+                     </div>
+                     <div data-bind="visible: page() == 'custom'" style="display: none;">
+                         % if not node['anonymous'] and 'admin' in user['permissions']:
+                            <a data-bind="click: showEditBox" class="pull-right"><i class="glyphicon glyphicon-pencil"></i> Edit</a>
+                         % endif
+
+                         <div class="m-b-md">
+                             <div class="citation-list">
+                                 <div class="row">
+                                     <div class="col-md-1">
+                                         <span id="custom-citation-copy-button" type="button" data-bind="attr: {'data-clipboard-text': customCitation}" class="btn btn-sm btn-default"><i class="fa fa-copy"></i></span>
+                                     </div>
+                                     <div class="col-md-9 m-l-sm">
```

Comment:
Changing these columns to xs (i.e. `col-xs-9`) makes this widget look a bit nicer on small screens. Currently it looks like:
![screen shot 2018-09-12 at 1 00 00 pm](https://user-images.githubusercontent.com/7913604/45441229-8a7e2680-b68c-11e8-8c12-84e8aa53716f.png)
but changing the columns to xs and moving the edit button into this row resizes like this:
![screen shot 2018-09-12 at 1 03 18 pm](https://user-images.githubusercontent.com/7913604/45441265-99fd6f80-b68c-11e8-9c70-5fac1b17db1b.png)
Here's the HTML from playing around in the console in case it's helpful: 
```
<div class="row">
	<div class="col-xs-1">
		<span id="custom-citation-copy-button" type="button" data-bind="attr: {'data-clipboard-text': customCitation}" class="btn btn-sm btn-default" data-clipboard-text="pls cite me like this"><i class="fa fa-copy"></i></span>
	</div>
	<div class="col-xs-9">
		<div class="f-w-xl">Cite as:</div>
		<span data-bind="text: customCitation">pls cite me like this</span>
	</div>
    <div class="col-xs-2"> 
		<a data-bind="click: showEditBox" class="pull-right"><i class="glyphicon glyphicon-pencil"></i> Edit</a>
	</div>
</div>
```

Reply:
It only happens on small screens. Switch your browser to mobile view and you should see what I mean.